music21/musicxml/m21ToXml.py

```diff
@@ -5477,6 +5485,46 @@ def articulationToXmlTechnical(self, articulationMark: articulations.Articulatio
         # mxArticulations.append(mxArticulationMark)
         return mxTechnicalMark
 
+    @staticmethod
+    def setBend(mxh: Element, bend: articulations.FretBend) -> None:
+        '''
+        Sets the bend-alter SubElement and the pre-bend,
+        release and with-bar SubElements when present.
+
+        Called from articulationToXmlTechnical
+
+        >>> MEXClass = musicxml.m21ToXml.MeasureExporter
+
+        >>> a = articulations.FretBend()
+
+        >>> from xml.etree.ElementTree import Element
+        >>> mxh = Element('bend')
+
+        >>> MEXClass.setBend(mxh, a)
+        >>> MEXClass.dump(mxh)
+        <bend>
+          <bend-alter></bend-alter>
+        </bend>
+        '''
+        bendAlterSubElement = SubElement(mxh, 'bend-alter')
+        alter = bend.bendAlter
+        if alter is not None:
+            # musicxml expects a number of semitones but not sure how to get it
+            # from a GeneralInterval
+            pass
+        if bend.preBend:
+            SubElement(mxh, 'pre-bend')
+        if bend.release is not None:
+            # Specifies where the release starts in terms of 
+            # divisions relative to the current note.
+            releaseSubElement = SubElement(mxh, 'release')
+            quarterLengthValue = bend.release
+            divisionsValue = defaults.divisionsPerQuarter * quarterLengthValue
```

Comment:
```
divisionsValue = int(defaults.divisionsPerQuarter * quarterLengthValue)
```

Musicxml "prefers" that offset be an integer:  
https://www.w3.org/2021/06/musicxml40/musicxml-reference/data-types/divisions/